web/src/settings_invites.ts

```diff
@@ -282,6 +298,9 @@ export function on_load_success(
         confirm_dialog.launch({
             html_heading: $t_html({defaultMessage: "Resend invitation?"}),
             html_body,
+            id: "resend_invite_modal",
+            close_on_submit: false,
+            loading_spinner: true,
```

Comment:
It can be fine with adding loading spinner in the next commit since client check should be quick, but this is fine too.